Resources/Private/Language/locallang.xlf

```diff
@@ -64,10 +64,6 @@
         <source>Please enter your password twice.</source>
       </trans-unit>
 
-      <trans-unit id="resetPassword.errors">
```

Comment:
Now you accidentally also dropped `validationErrors.errors`.

Reply:
Yes, i have re add this key now.